prisma/schema.prisma

```diff
@@ -82,5 +83,5 @@ model NFDEnjoyer {
   consecutiveFails  Int       @default(4)
   successfulMints   Int       @default(0)
   failedMints       Int       @default(0)
-  favourite         String    @default("")
+  favorites        String    @default("")
```

Comment:
Using a comma separate string is a pretty brittle way of handling arrays.
Usually in relational databases you create another table that connects the user and the dino [like you can see here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#explicit-many-to-many-relations) with the `CategoriesOnPosts` table.
Which allows you, instead of checking manually if the user has a dino as a favourite, to just check if there's a row with the `user id` and the `dino id` in that table and either add it or remove it.

Reply:
We have example of one-to-many already in our schema you could follow. See `duelStats` field from the User model and the `user` field from the Duels model